src/main/java/core/basesyntax/converter/impl/TypeConverter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.converter.impl;
+
+import core.basesyntax.converter.Convertor;
+import core.basesyntax.model.FruitTransaction;
+
+public class TypeConverter implements Convertor<FruitTransaction.Operation> {
+    @Override
+    public FruitTransaction.Operation convertor(String string) {
+        for (FruitTransaction.Operation operation : FruitTransaction.Operation.values()) {
+            if (operation.getTypeOfOperation().equals(string)) {
+                return operation;
+            }
+        }
+        return null;
```

Comment:
In my opinion, I don't need a separate class for this validation because I use it only once in my code.
What do you think about this?
As for the exception, I have added it.